Update firebase packages (#1118)
microsoft/appcenter-sdk-dotnet: CHANGELOG.md

# App Center SDK for .NET Change Log

## Version 2.4.0-preview (Under development)

### App Center

* **[Fix]** Fix application version being reported for ClickOnce deployments.

### App Center Push

#### Android

> * **[Breaking change]** Update Google Play Services' and Firebase's versions to v71 in Push module which forces applications to use Android 9.0 Mono frameworks.

Reviewer: ```suggestion
* **[Breaking change]** Update Google Play Services' and Firebase's versions to v71 in Push module which forces applications to use Android 9.0 Mono frameworks in build options. This has no impact on minimum Android SDK version which remains API level 16.
```

* **[Fix]** Fix application version being reported for ClickOnce deployments.

> #### Xamarin

Reviewer 2: ```suggestion
#### Android
```

* **[Fix]** Fix application version being reported for ClickOnce deployments.

> ### App Center Push

Reviewer 3: I think version should be bumped

Reviewer: Actually missing (Under development) for 2.4.0, it was not released.

#### Android

> * **[Breaking change]** We Updated Google Play Services and Firebase to v71 versions in Push module which forces applications to use Android 9.0 Mono frameworks.

Reviewer 3: ```suggestion
* **[Breaking change]** Update Google Play Services' and Firebase's versions to v71 in Push module which forces applications to use Android 9.0 Mono frameworks.
```

#### Android

> * **[Breaking change]** We Updated Google Play Services and Firebase to v71 versions in Push module which forces applications to use Android 9.0 Mono frameworks.

Reviewer: This is only in App Center Push, not in core.
There is not always a packages.config (s) file if you use project.json or modern csproj.
